Enterprise-CMCS/cmcs-eregulations
# EREGCSC-1552 / EREGCSC-1902 -- New Home Page Final Draft

Resolves [EREGCSC-1552](https://jiraent.cms.gov/browse/EREGCSC-1552) and [EREGCSC-1902](https://jiraent.cms.gov/browse/EREGCSC-1902)

**Description**

This PR combines the final two Homepage Update stories.  It will include the Search Resources CTA, as well as any changes/tweaks/fixes/etc to get the Homepage Update to a place where it can be shipped to users.

**This pull request changes:**

- adds a `?<YYYY-MM-DD>` param to the CSS `<link>` tag `href` in the base Django template
   - This will bust user's cached CSS file on their first visit of the day
   - see https://www.wkoorts.com/2009/07/12/css-auto-reload-with-django-templates/
- Adds "Try a Sample Search" CTA on the updated homepage
- CSS tweaks and fixes to get Homepage Update looking closer to Figma comp
- Overwrites all "old"/existing homepage templates and styles with new homepage template and styles
- Deletes all files/templates/views/etc associated with the `homepage_test` route

**Steps to manually verify this change:**

1. Visit the [Experimental Deployment](https://h04sv4nwg0.execute-api.us-east-1.amazonaws.com/dev844) and make sure the new homepage layout loads without issue
2. Visit [homepage_test on the Experimental Deployment](https://h04sv4nwg0.execute-api.us-east-1.amazonaws.com/dev844/homepage_test/) and make sure it doesn't exist
3. Normal site validation/smoke testing
